app/routes.py

```diff
@@ -1,2 +1,63 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, request, make_response
+
+planets_bp = Blueprint('planets', __name__, url_prefix="/planets")
+
+@planets_bp.route("", methods=["POST", "GET"])
+def handle_all_planets():
+  if request.method == "POST":
+    request_body = request.get_json()
+    new_planet = Planet(name=request_body["name"], description = request_body["description"], circum = request_body["circum"])
+    
+    db.session.add(new_planet)
+    db.session.commit()
+    return make_response(f"Planet {new_planet.name} create", 201)
+  
+  elif request.method == "GET":
+    
+    planets = Planet.query.all()
+    planets_response = []
+    for planet in planets:
+      planets_response.append({"id": planet.id, "name": planet.name, "description": planet.description, "circumference in mkm": planet.circum})
```

Comment:
We strongly suggest making a helper method on the Planet class to take care of converting the model instance to a dictionary for the purpose of being returned.

Reply:
For the keys in our json dictionaries, prefer a name that can be a valid variable name (generally, prefer _ to spaces).  While it doesn't impact our python handling that much, it can make the JSON handling for other languages (esp. JavaScript) if the keys can be treated as variables.